plugins/logs/plugin_test.go

```diff
@@ -20,26 +21,32 @@ import (
 	"github.com/open-policy-agent/opa/storage/inmem"
 )
 
-func TestPluginStart(t *testing.T) {
+func TestPluginStartRandomInput1(t *testing.T) {
```

Comment:
A couple high-level comments on these tests.

1. It's not obvious what each of these tests are exercising. What's the difference between RandomInput1, RandomInput2, and RandomInput3? I'd prefer a more descriptive name that makes the difference obvious.

2. I don't see a need to generate random data for these tests. In general, prefer to avoid using random data in these kinds of unit tests unless you really need randomness.

Reply:
Renamed the test names and removed the random input generation. Instead input is generated based on the test run index.